src/main/java/com/rummgp/medical_clinic/controller/PatientController.java

```diff
@@ -0,0 +1,46 @@
+package com.rummgp.medical_clinic.controller;
+
+import com.rummgp.medical_clinic.model.Patient;
+import com.rummgp.medical_clinic.service.PatientService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/patients")
+public class PatientController {
+    private final PatientService patientService;
+
+    @GetMapping
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Domyslnie dla odpowiedzi Sukcesu Spring dokleja status 200 (HttpStatus.OK) w przypadku jak chcemy aby nasz kontroler restowy zwracał status 200 to nie ma potrzeby definiowania tego @ResponseStatus(HttpStatus.OK)